Event service integration test flake

**What happened:**
The "integration" tests for the event service aren't very stable and often have to be re-run as we get an error:

```
=== RUN   TestEventService_Integration
    event_service_test.go:29: creating subscribers
    event_service_test.go:61: subscribers waiting for events
    event_service_test.go:58: finished publishing events
    event_service_test.go:94: rpc error: code = Canceled desc = context canceled
--- FAIL: TestEventService_Integration (0.01s)
```

**What did you expect to happen:**
I expect them to pass consistently


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- reignite version:
- OS (e.g. from `/etc/os-release`):

--- a/infrastructure/containerd/event_service_test.go
+++ b/infrastructure/containerd/event_service_test.go
@@ -7,7 +7,6 @@ import (
 	"time"
 
 	. "github.com/onsi/gomega"
-	"github.com/stretchr/testify/assert"
 
 	"github.com/weaveworks/reignite/api/events"
 	"github.com/weaveworks/reignite/core/ports"
@@ -106,8 +105,8 @@ func newSubscriber(t *testing.T, rootContext context.Context, data subData) {
 	recvd, err := watch(t, &subscriber, data.MaxEvents)
 	t.Logf("subscriber (%d) is done", data.ID)
 
-	assert.NoError(t, err)
-	assert.Len(t, recvd, data.MaxEvents)
+	Expect(err).To(BeNil())
+	Expect(recvd).To(HaveLen(data.MaxEvents))
 
 	data.Done()
 }

